controller/src/main/java/io/pravega/controller/server/security/auth/handler/impl/AclAuthorizerImpl.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server.security.auth.handler.impl;
+
+import io.pravega.auth.AuthHandler;
+import lombok.NonNull;
+
+class AclAuthorizerImpl extends AclAuthorizer {
+
+    @Override
+    public AuthHandler.Permissions authorize(@NonNull AccessControlList accessControlList, @NonNull String resource) {
+        AuthHandler.Permissions result = AuthHandler.Permissions.NONE;
+
+        String resourceDomain;
+        int indexOfPartsSeparator = resource.indexOf("::");
```

Comment:
what is the reason to use `:://` instead of `://` ?

Reply:
and should you simply split it by ":://" ?

Reply:
Actually, the resource looks like this: `prn::/scope:MarketData/stream:Prices`. There was an inaccurate comment elsewhere that right have misled you about the format.

More examples can be found [here](https://github.com/pravega/pravega/wiki/PDP-42:-New-Resource-String-Format-for-Authorization#examples).